.changelog/3323.breaking.md

```diff
@@ -0,0 +1,4 @@
+go: Move storage backend initialization to storage worker
+
+Notably, this also means that node command line options for storage
+configuration have been renamed (`--storage.*` to `--worker.storage.*`).
```

Comment:
Add a `cfg` changelog fragment and list the changes please.

Reply:
Also there's an additional configuration change that should also be listed: client's do not need to configure storage backed anymore